index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Sergio Usma's portfolio</title>
+  </head>
+  <body>
+    <div class="header">
+      <header>
+        <nav>
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+
+          <div class="burger">
+            <div class="burger-line top"></div>
+            <div class="burger-line mid"></div>
+            <div class="burger-line bot"></div>
+          </div>
+        </nav>
+      </header>
+    </div>
```

Comment:
- [OPTIONAL] I think it would be better to use `header` as the main container for the header instead of `div` ⚒ [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more information 💪
- [ ] Please consider using `nav` tag for hamburger menu for semantic reasons. The is because hamburger icon act as a menu for mobile devices 📱